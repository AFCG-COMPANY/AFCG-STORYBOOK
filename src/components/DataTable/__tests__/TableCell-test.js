import React from 'react';
import { mount } from 'enzyme';
import { Table, TableBody, TableRow, TableCell } from '../';

describe('DataTable.TableCell', () => {
  it('should render', () => {
    const wrapper = mount(
      <Table>
        <TableBody>
          <TableRow>
            <TableCell className="custom-class" />
          </TableRow>
        </TableBody>
      </Table>
    );
    expect(wrapper).toMatchSnapshot();
  });
});
