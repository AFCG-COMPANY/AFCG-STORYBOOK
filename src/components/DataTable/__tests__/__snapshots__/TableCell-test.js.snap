// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DataTable.TableCell should render 1`] = `
<Table>
  <table
    className="bx--data-table-v2 bx--data-table-v2--zebra"
  >
    <TableBody>
      <tbody>
        <TableRow>
          <tr>
            <TableCell
              className="custom-class"
            >
              <td
                className="custom-class"
              />
            </TableCell>
          </tr>
        </TableRow>
      </tbody>
    </TableBody>
  </table>
</Table>
`;
